protocols/relay/src/priv_client.rs

```diff
@@ -186,6 +192,10 @@ impl NetworkBehaviour for Behaviour {
                         .entry(peer_id)
                         .or_default()
                         .push(connection_id);
+
+                    let addr = endpoint.get_remote_address().clone();
```

Comment:
Instead of storing the remote address, it would be better if we'd use the address that the user gave us. You can get it from `TransportToBehaviourMsg::ListenReq`.

Reply:
I was wondering about if i should store it or obtain it from the event, but I agree it would be better to use what was given to us.